Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -912,9 +912,10 @@ def main():
             demisto.info(message)
             return_error(message)
             return
-    except LDAPSocketOpenError as e:
+    except Exception as e:
         exc_msg = str(e)
-        demisto.info(exc_msg)
+        demisto.info("Failed bind to: {}:{}. {}: {}".format(SERVER_IP, PORT, type(e), exc_msg
```

Comment:
this should be `demisto.error`

Reply:
Depends. I am ok with info (what it was also before) as it is in the flow of "test"